apps/trading-e2e/src/integration/assets.cy.ts

```diff
@@ -0,0 +1,177 @@
+    valueToolTip: 'An asset originated from an Ethereum ERC20 Token',
+  },
+  {
+    label: 'Name',
+    value: 'Euro',
+  },
+  {
+    label: 'Symbol',
+    value: 'tEURO',
+  },
+  {
+    label: 'Decimals',
+    value: '5',
+    labelTooltip: 'Number of decimal / precision handled by this asset',
+  },
+  {
+    label: 'Quantum',
+    value: '0.00001',
+    labelTooltip: 'The minimum economically meaningful amount of the asset',
+  },
+  {
+    label: 'Status',
+    value: 'Enabled',
+    labelTooltip: 'The status of the asset in the Vega network',
+    valueToolTip: 'Asset can be used on the Vega network',
+  },
+  {
+    label: 'Contract address',
+    value: '0x0158031158Bb4dF2AD02eAA31e8963E84EA978a4 ',
+    labelTooltip:
+      'The address of the contract for the token, on the ethereum network',
+  },
+  {
+    label: 'Withdrawal threshold',
+    value: '0.0005',
+    labelTooltip:
+      "The maximum you can withdraw instantly. There's no limit on the size of a withdrawal, but all withdrawals over the threshold will have a delay time added to them",
+  },
+  {
+    label: 'Lifetime limit',
+    value: '1,230.00',
+    labelTooltip:
+      'The lifetime deposit limit per address. Note: this is a temporary measure that can be changed or removed through governance',
+  },
+  { label: '', value: '' },
+  {
+    label: 'Infrastructure fee account balance',
+    value: '0.00001',
+    labelTooltip: 'The infrastructure fee account in this asset',
+  },
+  {
+    label: 'Global reward pool account balance',
+    value: '0.00002',
+    labelTooltip: 'The global rewards acquired in this asset',
+  },
+  {
+    label: 'Maker paid fees account balance',
+    value: '0.00003',
+    labelTooltip:
+      'The rewards acquired based on the fees paid to makers in this asset',
+  },
+  {
+    label: 'Maker received fees account balance',
+    value: '0.00004',
+    labelTooltip:
+      'The rewards acquired based on fees received for being a maker on trades',
+  },
+  {
+    label: 'Liquidity provision fee reward account balance',
+    value: '0.00005',
+    labelTooltip:
+      'The rewards acquired based on the liquidity provision fees in this asset',
+  },
+  {
+    label: 'Market proposer reward account balance',
+    value: '0.00006',
+    labelTooltip:
+      'The rewards acquired based on the market proposer reward in this asset',
+  },
+];
+//endregion
+
+beforeEach(() => {
+  cy.mockTradingPage();
+  cy.mockSubscription();
+  cy.setVegaWallet();
+});
+
+const visitPortfolioAndClickAsset = (assetName: string) => {
+  cy.visit('/#/portfolio');
+  cy.getByTestId(asset).contains(assetName).click();
+};
+
+const testTooltip = (index: number, testId: string, tooltip: string) => {
+  cy.getByTestId(`${index}_${testId}`).realHover();
+  cy.get('[role="tooltip"]').find('div').should('have.text', tooltip);
+  cy.getByTestId(dialogTitle).click();
+};
+
+describe('assets', { tags: '@smoke', testIsolation: true }, () => {
+  // 6501-ASSE-001
```

Comment:
suggestion (non blocking): I think it'd be better to add the comment next to the assertion it relates to

Reply:
Good Point, have made the change.